addons/website_sale/security/ir.model.access.csv

```diff
@@ -57,3 +57,5 @@ access_website_base_unit_public_public,website.base.unit public,model_website_ba
 access_website_base_unit_public_portal,website.base.unit public,model_website_base_unit,base.group_portal,1,0,0,0
 access_website_base_unit_public_employee,website.base.unit public,model_website_base_unit,base.group_user,1,0,0,0
 access_website_base_unit_sale_manager,website.base.unit sale manager,model_website_base_unit,sales_team.group_sale_manager,1,1,1,1
+access_website_checkout_step_restricted_editor,access_website_checkout_step restricted_editor,model_website_checkout_step,website.group_website_restricted_editor,1,0,0,0
```

Comment:
I don't think this should be needed, from what I've checked, the `restricted_editor` group don't have access to the option anyway.